### PR TITLE
fix: enable delete for active sessions

### PR DESCRIPTION
**Pull request by carissa-tang**
## Notion ticket link

[Enable deleting active test sessions](https://www.notion.so/uwblueprintexecs/Enable-deleting-active-test-sessions-a6a11c5dd81a48899824cbdf03c91989?pvs=4)

## Implementation description
* Add option to delete active test sessions in popover UI
* Update backend function and unit tests to support active sessions
* Delete `EmptyStudentsMessage.tsx` (unused)

## Steps to test
1. Create an upcoming and active test session. 
2. Verify that both test sessions you created have the delete option.
3. Verify that deleting is successful.
4. Verify that past test sessions do not have the delete option (and no popover in general)

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/services/implementations/__tests__/testSessionService.test.ts [hunk 1]
```diff
@@ -320,13 +320,26 @@ describe("mongo testSessionService", (): void => {
...
 
       await expect(async () => {
-        await testSessionService.deleteTestSession(savedTestSession.id);
+        await testSessionService.deleteTestSession(
+          savedTestSession.id,
+          new Date("2056-09-02T09:00:00.000Z"),
+        );
       }).rejects.toThrowError(
-        `Test Session id ${savedTestSession.id} not found or test session has already started`,
+        `Test Session id ${savedTestSession.id} not found or test session has past`,
```
Comment (jfdoming) [inline backend/services/implementations/__tests__/testSessionService.test.ts]: (non-blocking nit): past -> passed? (and other places)

**Review by jfdoming — APPROVED** [review]
clean

File: backend/services/implementations/__tests__/testSessionService.test.ts (hunk 1 above)
Comment (carissa-tang) [reply backend/services/implementations/__tests__/testSessionService.test.ts]: lol english hard ty

Comment (jfdoming) [reply backend/services/implementations/__tests__/testSessionService.test.ts]: 🥺 sorry for being annoying

**Pull request merged**